packages/block-editor/src/components/block-toolbar/style.scss

```diff
@@ -70,7 +70,7 @@
 	// Raise the specificity.
 	&.components-accessible-toolbar {
 		border: none;
-		border-bottom: $border-width solid $gray-200;
+		box-shadow: 0 $border-width 0 0 rgba($color: #000, $alpha: 0.133); // 0.133 = $gray-200 but with alpha.
```

Comment:
Do you know why this style is needed? I couldn't find any visual difference with or without this style.

Reply:
For mobile: 

![CleanShot 2024-08-28 at 16 47 35](https://github.com/user-attachments/assets/e72c4b9e-2027-4c6e-94b7-7ff15df1f69b)

Reply:
I see, thank you for the explanation.